.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
```

Comment:
@putmantime The build action. I merged this into `main` on my fork, ran the action, and it published a release which includes `merged_reasoned_loinc.owl`: https://github.com/joeflack4/comp-loinc/releases